### PR TITLE
First version of feature/progress bar added

### PR DESCRIPTION
**Pull request by amjadiftikhar**

### CODE REVIEW

File: src/client/components/ProgressBar/progress-bar.component.js [hunk 1]
```diff
@@ -1,105 +1,127 @@
...
+  const stageDaysArr = stageDays(stages);
+  const baseStage = [0];
+  const stageDiv = baseStage.concat(stageDaysArr);
+  const dayWidth = (1 / totalDays) * 100;
+  let stageNameIndex = 0;
+  const days = generateDays(totalDays);
+
+  const dayBlock = days.map((day) => {
+    const isPassed = day <= daysPassed;
+    const bgColor = isPassed ? 'skyblue' : 'rgb(204, 201, 201)';
```
[the remark this answers is not in the document]
Comment (amjadiftikhar) [reply src/client/components/ProgressBar/progress-bar.component.js]: Ok, we will try to fix it now.

[the remark this answers is not in the document]
Comment (senner008) [reply src/client/components/ProgressBar/progress-bar.component.js]: sounds promising :)

[the remark this answers is not in the document]
Comment (amjadiftikhar) [reply src/client/components/ProgressBar/progress-bar.component.js]: Done 👍

**Comment by senner007** [comment]
Is it ready for merge?

**Comment by humayunadilshahzad** [comment]
yes Niels ... it is ... we were just waiting for final comments ..if any ..other wise its complete

**Comment by amjadiftikhar** [comment]
> Is it ready for merge?


We hope so, Niels.

**Comment by senner007** [comment]
So did you find out what the problem was?

**Comment by amjadiftikhar** [comment]
> So did you find out what the problem was?

Yes I did, as we got help from Dan and then I pushed it again today. Niels, you are going to give any  feedback or someone else??

**Comment by senner007** [comment]
If you don't have merge problems and the code looks ok, then I suggest we merge now.

**Comment by amjadiftikhar** [comment]
> If you don't have merge problems and the code looks ok, then I suggest we merge now.

Should I delete this feature branch now? :)

**Comment by senner007** [comment]
Yes, just delete it

**Comment by senner007** [comment]
Remember for future reference, that you should only make changes to the relevant parts on the branch

**Pull request merged**